wandb/sdk/data_types/video.py

```diff
@@ -100,6 +102,15 @@ def __init__(
                 "wandb.Video accepts {} formats".format(", ".join(Video.EXTS))
             )
 
+        # TODO: Throw an error if fps is provided when passing raw bytes, or file path.
+        # For now display warning if fps is provided when passing raw bytes, or file path.
+        if isinstance(data_or_path, (BytesIO, str)) and fps:
+            with telemetry.context(obj=wandb.run._telemetry_obj) as tel:
```

Comment:
let's skip this, on a second thought as we won't be changing it regardless of usage

Reply:
also please don't access the internal telemetry object, it is also possible that at this point`wandb.run` doesn't even exists, so it would fail, because you will try to access an attribute on a null object!!!